yjit/src/yjit.rs

```diff
@@ -134,6 +134,13 @@ pub extern "C" fn rb_yjit_iseq_gen_entry_point(iseq: IseqPtr, ec: EcPtr, jit_exc
         return std::ptr::null();
     }
 
+    // If this is a large application (has very many ISEQs), switch to
+    // using the call threshold for large applications after this entry point
+    use crate::stats::rb_yjit_live_iseq_count;
+    if unsafe { rb_yjit_live_iseq_count } > LARGE_ISEQ_COUNT {
```

Comment:
```suggestion
    if rb_yjit_call_threshold < LARGE_CALL_THRESHOLD && unsafe { rb_yjit_live_iseq_count } > LARGE_ISEQ_COUNT {
```

Reply:
You're right that we shouldn't change the threshold if one is specified, but I think we should just check that it isn't the default value. I'll make that change 👍